From 4c387699b9da499e17000809d4a6f59170f4de48 Mon Sep 17 00:00:00 2001
From: Graeme Gregory <graeme@xora.org.uk>
Date: Wed, 18 Aug 2021 16:40:45 +0100
Subject: [PATCH 2/3] configure : alter /usr/local/qdos to qdos-gcc

To avoid conflicting with xtc68 which may have some
slight differences.

Signed-off-by: Graeme Gregory <graeme@xora.org.uk>
---
 configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure b/configure
index a7d3cb8..0c210b3 100755
--- a/configure
+++ b/configure
@@ -778,7 +778,7 @@ fi
 
 # Some systems (e.g., one of the i386-aix systems the gas testers are
 # using) don't handle "\$" correctly, so don't use it here.
-tooldir='$(exec_prefix)'/${target_alias}
+tooldir='$(exec_prefix)'/${target_alias}-gcc
 
 if [ "${host_alias}" != "${target_alias}" ] ; then
     if [ "${program_prefixoption}" = "" ] ; then
-- 
2.20.1

