diff --git a/ld/ldold.c b/ld/ldold.c
index 5909695..54f523b 100644
--- a/ld/ldold.c
+++ b/ld/ldold.c
@@ -1693,7 +1693,7 @@ void command_line(int   *xac, char  ***xav, char **paths, char *lib_arr)
        if(ldd == NULL)
        {
 #if defined(__unix__) || defined(__APPLE__)
-           ldd = strdup("/usr/local/qdos/lib/");
+           ldd = strdup("/usr/local/qdos-gcc/lib/");
 #else
            ldd = strdup("c:/qllib/");
 #endif
